source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure.Core/Cim/ValidationErrors/CimValidationErrorTextTemplateMessages.cs

```diff
@@ -133,6 +133,10 @@ public static class CimValidationErrorTextTemplateMessages
         public const string ChargeLinksUpdateNotYetSupportedErrorText =
             "Charge link for metering point ID {{MeteringPointId}} and Charge with ID {{DocumentSenderProvidedChargeId}} for owner {{ChargeOwner}} of type {{ChargeType}} cannot yet be updated or stopped. The functionality is not implemented yet";
 
+        [ErrorMessageFor(ValidationRuleIdentifier.SubsequentBundleOperationsFail)]
+        public const string SubsequentBundleOperationsFail =
+            "Transaction for Charge ID {{DocumentSenderProvidedChargeId}} is not completed, because a previous transaction for Charge ID {{TriggeredBySenderProvidedId}}, failed its validation";
```

Comment:
I would prefer the rejection message to go like this:

```suggestion
            "Transaction for Charge ID {{DocumentSenderProvidedChargeId}} is not completed: The request received contained multiple transactions for the same charge, and one of the previous transactions with ID {{TriggeredBySenderProvidedId}} failed validation why this transaction is also rejected";
```

Reply:
fixed